src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -318,6 +324,79 @@ def fuzz(self,
     return LibFuzzerCommon.fuzz(self, corpus_directories, fuzz_timeout,
                                 artifact_prefix, additional_args)
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner,LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+  def __init__(self, executable_path, default_args=None):
+    # This code assumes the following layout for files in the storage bucket:
+    # * multiboot.bin
+    # * fvm.blk
+    # * fuchsia-ssh.zbi
+    # * qemu/*
+    # * .ssh/*
+    local_path = os.getcwd() + "/"
+    resources_path = local_path + "fuchsia_on_clusterfuzz_resources_v1"
+
+    try:
+      os.mkdir(resources_path)
+    except OSError as e:
+      if e.errno == errno.EEXIST:
+        pass
+    subbed_fuchsia_gsutil_command = [param.replace("{fuchsia_resources_path}", environment.get_value('FUCHSIA_RESOURCES_PATH', ''))
```

Comment:
Why return an empty string from get_value if `FUCHSIA_RESOURCES_PATH` doesn't exist? I think this will cause the exception to manifest itself further from the root cause. You should probably log_error if `FUCHSIA_RESOURCES_PATH` doesn't exist instead.

Reply:
good point. added a log_error for that!